names.js

```diff
@@ -154,1119 +899,429 @@ const maleHumanNames = [
-"Savreimsen",
-"Lerototh",
-"Neneyin"
-];
+  "Zecria",
+  "Seiricyra",
+  "Fririssa",
+  "Eaxori",
+  "Afspira",
+  "Briwala",
+  "Lewala",
+  "Faith",
+  "Journey",
+  "Levdani",
+  "Levrali",
+  "Yapione",
+  "Dimzes",
+  "Kalrali",
+  "Pheborys",
+  "Yorasolis",
+  "Interesting",
+  "Song",
+  "Darkness",
+  "Zaseis",
+  "Nithki",
+  "Kalmaia",
+  "Yorabis",
+  "Pespunith",
+  "Zaipunith",
+  "Levyis",
+  "Timeless",
+  "Gloom",
+  "Chant",
+  "Stresral",
+  "Lennel",
+  "Shosith",
+  "Valdin",
+  "Thilah",
+  "Thorzendes",
+  "Khordradrel",
+  "Savreimsen",
+  "Lerototh",
+  "Neneyin"
+  ];
 
+// MALE
 const maleTieflingNames = [
-"Hormos",
-"Nephrut",
-"Malvir",
-"Valrus",
-"Uriira",
-"Karai",
-"Lokelech",
-"Eternal",
-"Rolnon",
-"Valthus ",
-"Meichar",
-"Thynemenos",
-"Kilrus",
-"Xarmos",
-"Dharron",
-"Salrius",
-"Valakas",
-"Zhermos",
-"Malenon",
-"Zorlius ",
-"Barilius",
-"Mavchar",
-"Nephmeros",
-"Killius",
-"Ekreus",
-"Carlech",
-"Arrai ",
-"Dokuc",
-"Yinry",
-"Adriath",
-"Yicor",
-"Zusuc",
-"Areci",
-"Agecus",
-"Elladrun",
-"Hilliardan",
-"Ertiallat"
-];
+  "Hormos",
+  "Nephrut",
+  "Malvir",
+  "Valrus",
+  "Uriira",
+  "Karai",
+  "Lokelech",
+  "Eternal",
+  "Rolnon",
+  "Valthus ",
+  "Meichar",
+  "Thynemenos",
+  "Kilrus",
+  "Xarmos",
+  "Dharron",
+  "Salrius",
+  "Valakas",
+  "Zhermos",
+  "Malenon",
+  "Zorlius ",
+  "Barilius",
+  "Mavchar",
+  "Nephmeros",
+  "Killius",
+  "Ekreus",
+  "Carlech",
+  "Arrai ",
+  "Dokuc",
+  "Yinry",
+  "Adriath",
+  "Yicor",
+  "Zusuc",
+  "Areci",
+  "Agecus",
+  "Elladrun",
+  "Hilliardan",
+  "Ertiallat"
+  ];
 
+// SURNAME
 const tieflingSurnames = [
-"Vervend ",
-"Brotaebae ",
-"Tovos ",
-"Zegraki ",
-"Shama ",
-"Toldris ",
-"Rova ",
-"Ecorvot ",
-"Drardim ",
-"Shagrerat ",
-"Mokot ",
-"Nirvad ",
-"Sashakeand ",
-"Fondre ",
-"Letevri ",
-"Veni ",
-"Gibon",
-"Dobramia",
-"Nende",
-"Zetedaand",
-"Harvo",
-"Pasrossa",
-"Abem",
-"Riavagrat",
-"Fommeld",
-"Drorhiki"
- ];
+  "Vervend ",
+  "Brotaebae ",
+  "Tovos ",
+  "Zegraki ",
+  "Shama ",
+  "Toldris ",
+  "Rova ",
+  "Ecorvot ",
+  "Drardim ",
+  "Shagrerat ",
+  "Mokot ",
+  "Nirvad ",
+  "Sashakeand ",
+  "Fondre ",
+  "Letevri ",
+  "Veni ",
+  "Gibon",
+  "Dobramia",
+  "Nende",
+  "Zetedaand",
+  "Harvo",
+  "Pasrossa",
+  "Abem",
+  "Riavagrat",
+  "Fommeld",
+  "Drorhiki"
+   ];
 
-const maleGnomeNames = [
-"Trutt",
-"Zod",
-"Cant",
-"Vriucdien",
-"Shmuakqa",
-"Puthatt",
-"Ivit",
-"Gobritan",
-"Kokruakrast",
-"Rarickembitt"
-];
 
-const femaleGnomeNames = [
-"Vock",
-"Huck",
-"Zuth",
-"Nus",
-"Haxis",
-"Rulbush",
-"Ikzuth",
-"Utrothraim",
-"Zosuffosh",
-"Sensannailzish"
-];
-
-const gnomeSurnames = [
-"Dodlawams",
-"Hasp",
-"Epswodnith",
-"Klippiesp",
-"Qrondiornficks",
-"Ter",
-"Pedomber",
-"Bems",
-"Uplizbiss",
-"Qagnom"
-];
 
-const maleHalflingNames = [
-"Lal",
-"Dar",
-"Dord",
-"Ximoth",
-"Morrith",
-"Vruraard",
-"Turind",
-"Trasgaantard",
-"Hedirrorn",
-"Sarromrir"
-];
+/////////////* SWITCH FUNCTION */////////////
```

Comment:
I can't see the switch function in the review, is that because it hasn't changed?

Reply:
Nevermind if i view the file I can see it's still there :)